src/lib/tool_providers.ml

```diff
@@ -468,13 +461,33 @@ let muse_tool ~host ~meta_playground =
                Program.(
                  exec ["mkdir"; "-p"; install_path]
                  && exec ["cd"; install_path]
-                 && download_url_program url
+                 && Workflow_utilities.Download.wget_program url
                  && shf "chmod +x %s" binary_path
                ))
   in
   let init = Program.(shf "export muse_bin=%s" binary_path) in
   Tool.create Tool.Default.muse ~ensure ~init
 
+let opam_tool ~host ~meta_playground =
+  let open KEDSL in
+  let url =
+    "https://github.com/ocaml/opam/releases/download/1.2.2/opam-1.2.2-x86_64-Linux"
+  in
+  let install_path = meta_playground // "opam" in
+  let binary_path = install_path // "opam" in
+  let ensure =
```

Comment:
Ah, good point, this was from the first pass. I'll get rid of it as a tool in `Tool_providers` entirely.